src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Build/OutputGroup.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Build
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Diagnostics;
```

Comment:
We have usings outside of the namespace. I'll update our .editorconfig to express this.